tools/clang/lib/Sema/SemaType.cpp

```diff
@@ -4538,11 +4531,9 @@ static void fillAttributedTypeLoc(AttributedTypeLoc TL,
 
   // HLSL changes begin
   // Don't fill the location info for matrix orientation attributes
-  if (!attrs && !DeclAttrs) {
```

Comment:
This change should fix #1805.

Reply:
Oh nice, that's some real-time bug fixing! Thanks :)